### PR TITLE
Fix a small typo in GC example

### PR DESCRIPTION
**Pull request by Rishit-dagli**
I noticed a really small typo in the the [Gradient Centralization for Better Training Performance](https://keras.io/examples/vision/gradient_centralization/) example and this PR fixes that.

Earlier it was (notice spacing):
> We will rescale the data to [0, 1]  andperform simple augmentations to our data.

Which this PR changes to:
> We will rescale the data to [0, 1] and perform simple augmentations to our data.

### CODE REVIEW

**Review by fchollet — APPROVED** [review]
LGTM, thanks
